db/migrate/20250312090252_create_organisations.rb

```diff
@@ -0,0 +1,15 @@
+class CreateOrganisations < ActiveRecord::Migration[8.0]
+  def change
+    create_table :organisations do |t|
```

Comment:
Would really love us to use UUIDs as Ids across the board

Reply:
I'll open a follow on PR